automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -81,5 +87,88 @@ class RawlsApiSpec extends FreeSpec with Matchers with CleanUp with BillingFixtu
         }
       }
     }
+
+    object SingleParticipant {
+      val participantEntity = "entity:participant_id\nparticipant1"
```

Comment:
and the most minimal TSV award goes to... :trophy:

Reply:
stole this from somewhere